tests/testTriggers.go

```diff
@@ -5907,6 +5911,81 @@ func TriggerAsyncDRVolumeOnly(contexts *[]*scheduler.Context, recordChan *chan *
 	updateMetrics(*event)
 }
 
+func TriggerStorkApplicationBackup(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	dash.Infof("Stork Appplication Backup triggered at: %v", time.Now())
+	defer endLongevityTest()
+	startLongevityTest(StorkApplicationBackup)
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: AppTasksDown,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	setMetrics(*event)
+	chaosLevel := ChaosMap[StorkApplicationBackup]
+
+	var (
+		s3SecretName     = "s3secret"
+		backupNamespaces []string
+		timeout          = 5 * time.Minute
+		taskNamePrefix   = "stork-app-backup-"
+	)
+
+	Step(fmt.Sprintf("Deploy applications for for backup, with frequency: %v", chaosLevel), func() {
+
+		// Write kubeconfig files after reading from the config maps created by torpedo deploy script
+		err := asyncdr.WriteKubeconfigToFiles()
+		if err != nil {
+			dash.Errorf("Failed to write kubeconfig: %v", err)
+		}
+
+		err = SetSourceKubeConfig()
+		if err != nil {
+			dash.Errorf("Failed to Set source kubeconfig: %v", err)
+		}
+		UpdateOutcome(event, err)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d-%s", taskNamePrefix, i, time.Now().Format("15h03m05s"))
+			dash.Infof("Task name %s\n", taskName)
+			appContexts := ScheduleApplications(taskName)
+			*contexts = append(*contexts, appContexts...)
+			ValidateApplications(*contexts)
+			for _, ctx := range appContexts {
+				// Override default App readiness time out of 5 mins with 10 mins
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				backupNamespaces = append(backupNamespaces, namespace)
+			}
+		}
+		dash.Infof("Backup applications, present in namespaces - %v", backupNamespaces)
+		dash.Infof("Start backup application")
+		for i, currbkNamespace := range backupNamespaces {
+			taskNamePrefix = taskNamePrefix + fmt.Sprintf("%d", i)
+			currBackupLocation, err := applicationbackup.CreateBackupLocation(taskNamePrefix+"-location", currbkNamespace, s3SecretName)
+			if err != nil {
+				UpdateOutcome(event, fmt.Errorf("backup location creation failed with %v", err))
+			}
+			bkp, bkp_create_err := applicationbackup.CreateApplicationBackup(taskNamePrefix+"-backup", currbkNamespace, currBackupLocation)
```

Comment:
if there is an error in ceating backup location, do you still want to continue with backup creation, if not can we just return from here

Reply:
Done